### DISCUSSION TITLE
Require recommendation from developer

### DISCUSSION THREAD

**Discussion by Universalizer** [Divested-Mobile/Mull-Fenix#154] [Show and tell]
Hello Developer, 
https://github.com/Gedsh/InviZible/wiki/InviZible---Tor-and-Firefox-browser

Can i use Mull in these way ?

Your recommendation ! Please.

**Comment by SkewedZeppelin** [comment]
You can, but I do not recommend it.

Either use Tor Browser
or
Orbot full device VPN + kill switch

**Reply by Universalizer** [reply]
Due to Tor browser has some problems,

Orbot full device VPN + kill switch, Alright.

Gratitude & Respect.
